CMakeLists.txt

```diff
@@ -1,14 +1,14 @@
 cmake_minimum_required (VERSION 3.10)
-project(parakeet_ros VERSION 1.0.0)
+project(parakeet_ros VERSION 1.0.1)
 
 ## Find catkin and any catkin packages
 find_package(catkin REQUIRED COMPONENTS roscpp rospy std_msgs genmsg sensor_msgs)
 
 ## Generate added messages and services
-generate_messages(DEPENDENCIES std_msgs)
+#generate_messages(DEPENDENCIES std_msgs)
```

Comment:
Why was this commented out?

Reply:
std_msgs is declared as a dependency to the ROS project. This line in CMakeLists and the dependency in ROS have the same effect. Therefore, it is not needed. Why I chose to comment it instead of removing, I'm not sure. I'll go in and remove the comment.

Reply:
Comment removed.